docker-compose.yml

```diff
@@ -150,7 +150,79 @@ services:
       - traefik.docker.network=cryptlex-on-premise_backend
       - traefik.port=4200
 
-  release-server:
+ admin-portal:
+    image: cryptlex/admin-portal:latest
+    depends_on:
+      - web-api
+    env_file: web-portals.env
+    environment:
+      WEB_API_BASE_URL: https://${WEB_API_DOMAIN}
+      RELEASE_SERVER_BASE_URL: https://${RELEASE_SERVER_DOMAIN}
+
+    networks:
+      - backend
+    restart: unless-stopped
+    ports:
+      - 4400
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "20m"
+    labels:
+      - traefik.backend=cryptlex/cryptlex-admin-portal
```

Comment:
please clarify why backend uses this value? Have we used this pattern anywhere in the file